cli/src/conf/conf_get.rs

```diff
@@ -0,0 +1,64 @@
+use crate::{command_title, prelude::*};
+
+/// Print the current Nexus CLI configuration.
+pub(crate) async fn print_nexus_conf(conf_path: PathBuf) -> AnyResult<String, NexusCliError> {
+    let conf = CliConf::load_from_path(&conf_path)
+        .await
+        .unwrap_or_default();
+
+    command_title!("Current Nexus CLI Configuration");
+
+    toml::to_string_pretty(&conf).map_err(|e| {
```

Comment:
TOML is not JSON, and generally here we do not respect the --json arg.

E.g. run:

```shell
$ nexus conf get --json
[sui]
net = "Localnet"
wallet_path = "/home/user/.sui/sui_config/client.yaml"
```

which is TOML but not JSON.

Apparently, all the other variations work and give the same results:
```shell
$ nexus conf --json get
[sui]
net = "Localnet"
wallet_path = "/home/user/.sui/sui_config/client.yaml"

$ nexus --json conf get
[sui]
net = "Localnet"
wallet_path = "/home/user/.sui/sui_config/client.yaml"
```


JSON handling perhaps needs an overhaul and deserves a separate ticket. It's going to be a bit tedious to go over everything but we should.

Reply:
https://github.com/Talus-Network/nexus-sdk/issues/175

Feel free to resolve